### ISSUE TITLE
dod is always null

### ISSUE THREAD

**Issue by arielhasidim** [MIT-LCP/mimic-iv#103]
Hello,
dod column in mimic_core->patients is always null.
check:
```
SELECT * FROM `physionet-data.mimic_core.patients`
where dod is not null
```
returns: `This query returned no results.`

BTW:
In the documentation I read that _"Date of death is extracted from the hospital information system only, and does not include out of hospital mortality as of MIMIC-IV v1.0."_. First-of-all this is a real shame. Secondly, if this is the new reallty, maybe `patients.dod` should be dropped completely, and users should use deathtime from admission table only.

**Comment by alistairewj** [comment]
Duplicate of #71.

Unfortunately the lack of out-of-hospital death data is due to the change in the Social Security Death Master File (SSDMF) which resulted in it no longer capturing deaths in state databases. This resulted in a ~40% "drop" in out of hospital mortality, due to the missing data. The data are too poor quality to include. We've looked at other sources - in particular the CDC is an option - but it is very expensive.

**Comment by alistairewj** [comment]
P.S. This will be fixed in v1.0.
